lib/rack/prx_auth/token_data.rb

```diff
@@ -22,13 +24,25 @@ def user_id
       end
 
       def authorized?(resource, scope=nil)
+        return globally_authorized?(scope) if resource == WILDCARD_RESOURCE_NAME
+
+        authorized_for_resource?(resource, scope) || (scope.nil? ? false : globally_authorized?(scope))
+      end
+
+      def globally_authorized?(scope)
```

Comment:
So are you imagining feed would do something like this?

```ruby
if globally_authorized?("read-private")
  Episode.all
else
  Episode.where("account_id IN ?", ids)
end
```

Wondering if "read-private" is what we'll use on day 1 here.  And also, does this expect a symbol or string for the scope?

Reply:
(Wondering if that'll be weird since there's a dash in it).

Reply:
Looks ~ right to me! Longer term, I would advocate for even more fine-grained scopes (e.g. feeder-read-unpublished-episodes) but this seems right to me.

Strings are permitted for sure, this is dealing with strings under the hood. Maybe it would make sense to make the scopes more easily symbolized? Or change the tests to make clear that strings are cool.

Reply:
Seems like normalizing to lowercase/underscore might make sense - belt and suspenders!  So `authorized(:read_private) == authorized('read_private') == authorized('read-private')`.

But can iron some of that out later!